modules/core/src/main/java/org/apache/synapse/util/ConcurrencyThrottlingUtils.java

```diff
@@ -47,9 +47,9 @@ public static void decrementConcurrencyThrottleAccessController(MessageContext s
         if (isConcurrencyThrottleEnabled != null && isConcurrencyThrottleEnabled) {
             ConcurrentAccessController concurrentAccessController = (ConcurrentAccessController) synCtx
                     .getProperty(SynapseConstants.SYNAPSE_CONCURRENT_ACCESS_CONTROLLER);
+            int available = concurrentAccessController.incrementAndGet();
+            int concurrentLimit = concurrentAccessController.getLimit();
             if (log.isDebugEnabled()) {
-                int available = concurrentAccessController.incrementAndGet();
-                int concurrentLimit = concurrentAccessController.getLimit();
                 log.debug("Concurrency Throttle : Connection returned" + " :: " + +available + " of available of"
```

Comment:
There are two + signs and space is missing at the end of "of available of". Please correct them as well

Reply:
+1 Done

Reply:
@GDLMadushanka you might have missed adding " of available of" space and make this " of available of " ?